client/my-sites/domains/domain-management/primary-domain/index.jsx

```diff
@@ -96,32 +97,29 @@ class PrimaryDomain extends React.Component {
 
 		return (
 			<Main className="domain-management-primary-domain">
+				/* eslint-enable wpcalypso/jsx-classname-namespace */
```

Comment:
there's one left which can be removed I think

Reply:
Oops, I missed that one. 🤦🏽‍♂️

I rebased with master before pushing the last commit 53079dc. Would a `force-push` be required for all commits going forward?

Reply:
I'm not sure I can follow. I think you can add another commit which removes this line, would that be sufficient?

Alternatively, you can add this change to the git staging area (`git add <file>`) and use `git commit --amend` to add those changes to the latest commit. There a force-push is required  because it changes the git history (the commit hash).

Reply:
Yes, looks like it was okay to not use the `--force` flag on 2da1d05. 👍🏽 ✅